test/unit/murmurhash3_spec.js

```diff
@@ -0,0 +1,53 @@
+/* globals jasmine, expect, it, describe, MurmurHash3_64 */
+'use strict';
+
+describe('MurmurHash3_64', function() {
+
+  it('instantiates without seed ', function() {
```

Comment:
Nit: remove the space at the end of the string. This also applies to the unit test below.